.github/main.workflow

```diff
@@ -0,0 +1,28 @@
+workflow "Build and publish" {
+  on = "push"
+  resolves = ["publish"]
+}
+
+action "install" {
+  uses = "actions/npm@59b64a598378f31e49cb76f27d6f3312b582f680"
+  args = "install"
+}
+
+action "build" {
+  uses = "actions/npm@59b64a598378f31e49cb76f27d6f3312b582f680"
+  needs = ["install"]
+  args = "run build"
+}
+
+action "on master branch" {
+  uses = "actions/bin/filter@3c0b4f0e63ea54ea5df2914b4fabf383368cd0da"
+  needs = ["build"]
```

Comment:
It looks to me like this will do the build (and therefore the install) on every branch, and then only publish if on the master branch. Is that
1. accurate
2. intended

Reply:
Yep, to both. We should know if a pull request breaks the build before we merge it.